tomato-timer/src/Memo.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import './App.css';
+import * as firebase from 'firebase';
+
+
+
+class Memo extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            memo: "",
+            tasks: [],
+        }
+    }
+
+    handleMemoSubmit(e) {
+        var updatedTaskList = this.state.tasks;
+
+        updatedTaskList.push(this.state.memo);
+
+        this.setState({
+            memo: "",
+            tasks: updatedTaskList
+        })
+
+        console.log(updatedTaskList);
+
+        // this.sendData(updatedTaskList, e);
```

Comment:
never commit commented out code, you can always revert changes later if you make small, sensible commits with good messages.